src/web/components/Nav/Nav.tsx

```diff
@@ -65,12 +66,50 @@ const PositionButton = ({ children }: { children: React.ReactNode }) => (
     </div>
 );
 
-export const Nav = ({ display, pillar, nav, subscribeUrl, edition }: Props) => {
-    const [showExpandedMenu, toggleExpandedMenu] = useState<boolean>(false);
-    const mainMenuId = 'main-menu';
+// The checkbox ID is used as a CSS selector to enable NoJS support
+const menuCheckboxId = 'main-menu-toggle';
```

Comment:
I think we've evolved past the point where these variables are needed. We can just inline the text